data.md

```diff
@@ -237,10 +245,13 @@ Bitwise Operators
                  | twos ( Int , Int )       [function]
                  | bswap ( Int , Int )      [function]
  // --------------------------------------------------
-    rule signextend(N, W) => twos(N +Int 1, W) -Int (1 <<Byte (N +Int 1))  requires         word2Bool(bit((8 *Int (N +Int 1) -Int 1), twos(N +Int 1, W)))
-    rule signextend(N, W) => twos(N +Int 1, W)                             requires notBool word2Bool(bit((8 *Int (N +Int 1) -Int 1), twos(N +Int 1, W)))
+    rule signextend(N, W) => twos(N, W) -Int (1 <<Byte N)  requires N>Int 0 andBool        word2Bool(bit((8 *Int N -Int 1), twos(N, W)))
+    rule signextend(N, W) => twos(N, W)                    requires N>Int 0 andBool notBool word2Bool(bit((8 *Int N -Int 1), twos(N, W)))
+    rule signextend(0, _) => 0
```

Comment:
Is there no documentation change in `Design.md` because the previous implementation was incorrect?

Reply:
yes, it looks like the docs were out of sync previously.